### PR TITLE
feat(tianmu): to support replace into values() SQL synax (#1011)

### LINKED ISSUE #1011: feature: to support ` replace into  values()` SQL synax
**Is your feature request related to a problem? Please describe.**

To support ` replace into  values()` SQL synax.
For example: REPLACE INTO test VALUES(1,'Old','2014-08-20 18:47:00');.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by isredstar**
## Summary about this PR

Issue Number: close #1011 

## Tests Check List

- [x] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

## Changelog

- [x] New Feature
- [ ] Bug Fix
- [ ] Performance Improvement
- [ ] Build/Testing/CI/CD
- [ ] Documentation
- [ ] Not for changelog (changelog entry is not required)

## Documentation

- [x] Affects user behaviors
- [x] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features

### CODE REVIEW

File: mysql-test/suite/tianmu/t/replace.test [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+drop table if exists test1;
+--enable_warnings
+# test auto_increment and the primary key covers 1 columns
+CREATE TABLE test1 (
+  id INT NOT NULL AUTO_INCREMENT,
+  data VARCHAR(64) DEFAULT NULL,
+  ts TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  PRIMARY KEY (id)
+)  engine=tianmu;
+REPLACE INTO test1 VALUES (1, 'Old', '2014-08-20 18:47:00');
```
Comment (RingsC) [inline mysql-test/suite/tianmu/t/replace.test:17]: Pls, add a test like:
```
insert into test1 values (1,xxx);
insert into test1 values (2, xxxx);
replace into test1 values(1, xxxx);
delete from test1 where id =1;
replace int test1 (1,xxxx);
```
Thanks. 



```

**Review by RingsC — CHANGES_REQUESTED** [review]

File: mysql-test/suite/tianmu/t/replace.test (hunk 1 above)
Comment (isredstar) [reply mysql-test/suite/tianmu/t/replace.test:17]: Thank for your advice. I'll make the requested changes!

Comment (isredstar) [reply mysql-test/suite/tianmu/t/replace.test:17]: I have done the requested changes, please review again;

```
# test whether `delete` would impact the `replace into` action
--disable_warnings
drop table if exists test1;
--enable_warnings
CREATE TABLE test1 (
  id INT NOT NULL AUTO_INCREMENT,
  data VARCHAR(64) DEFAULT NULL,
  ts TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
  PRIMARY KEY (id)
)  engine=tianmu;
insert into test1 values (1,'Old', '2014-08-20 18:47:00');
insert into test1 values (2, 'Old', '2014-08-20 18:47:00');
REPLACE INTO test1 VALUES (1, 'New', '2014-08-20 18:47:42');
SELECT * FROM test1;
delete from test1 where id =1;
SELECT * FROM test1;
REPLACE INTO test1 VALUES (1, 'NewNew', '2014-08-20 18:47:55');
SELECT * FROM test1;
REPLACE INTO test1 VALUES (1, 'NewNewNew', '2014-08-20 18:55:55');
SELECT * FROM test1;
drop table test1;
```

**Review by lujiashun — APPROVED** [review]
LGTM

**Review by duanfuxiang0 — APPROVED** [review]
LGTM

**Review by RingsC — APPROVED** [review]

**Pull request closed without merging**